kbase/narrative
# Dataup 224 import button

# Description of PR purpose/changes

![Screen Shot 2020-10-25 at 9 16 48 PM](https://user-images.githubusercontent.com/4184019/97133487-79c38a00-1707-11eb-8d77-c410d3cca0ee.png)

* The most recent edition in this ticket, with the import button implemented and functionality hooked up. For now we just initialize individually a new instance of each import app cell that has been selected. So if a user uploaded 5 SRA reads files and selected them all to import, they would have 5 app cells. This should be able to be easily changed as we work through bulk importer cells. 

* I also did some more refactoring of stagingAreaViewer, added some accessibility features. 
* The main feature branch has been updated with latest from Truss to include all of the CSS changes, and this branch includes those as well. 

* The remaining pieces on this ticket are to polish off any visual tweaks (if you see them please give feedback) and to implement tests. 

* Once this one gets approved I will merge to the main DATAUP-224 feature branch, implement tests, do one more PR review and then hopefully we should be g2g 🤞 

# Jira Ticket / Issue #
https://kbase-jira.atlassian.net/browse/DATAUP-224

# Testing Instructions
* Pull down this branch and navigate to the import tab of the data panel slide out.
* Add files if you don't already have some
* Checkboxes on each row should be disabled until the user selects a type
* As soon as one data type is selected the select all checkbox is also enabled
* As soon as a single (or all) checkbox(es) is(are) selected the Import Selected button is enabled
* Disabled Import Selected button has tooltip "Please select a file/s to continue."
* On import a new app cell is added for each checked entry

# Dev Checklist:
